### PR TITLE
Bugfix/data standards update

### PR DESCRIPTION
**Pull request by Djangofs**

### CODE REVIEW

File: openAPI.spec.json [hunk 1]
```diff
@@ -232,6 +244,24 @@
...
+          "type": "string",
+          "description": "The local council the registration belongs to"
+      },
+      "collected": {
+        "type": "boolean",
+        "description": "Flag indicating whether this registration has been collected by the council or not"
+      },
+      "collected_at": {
+        "type": "string",
+        "description": "The date and time the registratio was collected"
```
Comment (off-beat) [inline openAPI.spec.json]: "time that the registration..."

File: openAPI.spec.json [hunk 2]
```diff
@@ -232,6 +244,24 @@
...
       },
+      "fsa_rn": {
+          "type": "string",
+          "maximum": 20,
+          "minimum": 20,
+          "description": "The unique reference number to identify this registration"
+      },
+      "council": {
+          "type": "string",
+          "description": "The local council the registration belongs to"
```
Comment (off-beat) [inline openAPI.spec.json]: "**that t**he reg..."

File: openAPI.spec.json (hunk 1 above)
Comment (off-beat) [reply openAPI.spec.json]: I'm doubting my grasp of grammar now. I guess technically "time at which the...." but probably being pedantic now

File: openAPI.spec.json [hunk 3]
```diff
@@ -253,115 +283,208 @@
...
+                  "type": "string",
+                  "required": true,
+                  "maximum": "255",
+                  "description": "Uses [npm Validator](https://www.npmjs.com/package/validator) isEmail"
               },
               "establishment_opening_date": {
-                  "type": "string"
+                  "type": "string",
+                  "required": true,
+                  "description": "Uses [npm Validator](https://www.npmjs.com/package/validator) isISO8601"
```
Comment (off-beat) [inline openAPI.spec.json]: should be also just state the format to be clear? e.g. "YYYY-MM-DD - Uses [npm Val...."

File: openAPI.spec.json [hunk 4]
```diff
@@ -253,115 +283,208 @@
...
-                  "type": "string"
+                  "type": "string",
+                  "required": true,
+                  "description": "Possible values: 'Other businesses', 'End consumer', 'End consumer and other businesses'"
               },
               "business_type": {
-                  "type": "string"
+                  "type": "string",
+                  "required": true,
+                  "description": "Predetermined values, see separate document for full business type list"
```
Comment (off-beat) [inline openAPI.spec.json]: do they have this document? Maybe it's not so relevant to them I don't know.

**Comment by off-beat** [comment]
I get a few issues when validating schema here: https://apidevtools.org/swagger-parser/online/

File: openAPI.spec.json (hunk 4 above)
Comment (Djangofs) [reply openAPI.spec.json]: We will make this document available as well in fact is already on GitHub

**Pull request merged**